README.md

```diff
@@ -1,46 +1,161 @@
+| `sympa_db_port`                            | `3306`                                                                                                                | Port of the database server                                                                                                                                                                                                                                                                                                              |
+| `sympa_db_user`                            | `sympa`                                                                                                               | User for the database connection  User for the database connection  User for the database connection                                                                                                                                                                                                                                     |
+| `sympa_lang`                               | `en`                                                                                                                  | Default language (one of supported languages)#supported_lang	ca,cs,de,el,es,et,en-US,fr,fi,hu,it,ja,ko,nl,nb,oc,pl,pt-BR,ru,sv,tr,vi,zh-CN,zh-TW                                                                                                                                                                                      |
+| `sympa_logo_html_definition`               | :heavy_multiplication_x:                                                                                              | HTML snippet to place logo in upper left corner                                                                                                                                                                                                                                                                                          |
+| `sympa_gecos`                              | `SYMPA`                                                                                                               | Gecos for service mail sent by Sympa itself.                                                                                                                                                                                                                                                                                             |
+| `sympa_process_archive`                    | `"off"`                                                                                                               | Store distributed messages into archive                                                                                                                                                                                                                                                                                                  |
```

Comment:
Why is this a string and not a boolean flag? If only `on` and `off` are possiblem, this should be abstracted using Ansible. Same for other variables.